src/meshes/mesh_io.jl

```diff
@@ -97,7 +97,8 @@ end
 # of the mesh, like its size and the type of boundary mapping function.
 # Then, within Trixi2Vtk, the StructuredMesh and its node coordinates are reconstructured from
 # these attributes for plotting purposes
-function save_mesh_file(mesh::StructuredMesh, output_directory; system = "")
+function save_mesh_file(mesh::StructuredMesh, output_directory; system = "",
+                        timestep = 0)
```

Comment:
```suggestion
function save_mesh_file(mesh::StructuredMesh, output_directory; system = "")
```
unused?

Reply:
Otherwise, if it is needed for compatibility, it would be good imho to add a short comment stating as much
```suggestion
# Note: the `timestep` argument is needed for compatibility with the method for
# `StructuredMeshView`
function save_mesh_file(mesh::StructuredMesh, output_directory; system = "",
                        timestep = 0)
```

Reply:
If I remember correctly I added this for compatibility. So, better keep it.